src/Compilers/CSharp/Portable/Symbols/Source/SourceOrdinaryMethodSymbol.cs

```diff
@@ -1034,6 +1034,9 @@ internal override void AfterAddingTypeMembersChecks(ConversionsBase conversions,
             }
 
             ParameterHelpers.EnsureIsReadOnlyAttributeExists(Parameters, diagnostics, modifyCompilationForRefReadOnly: true);
+
+            this.EnsureNullableAttributeExistsIfNecessary(ReturnType, diagnostics);
+            ParameterHelpers.EnsureNullableAttributeExistsIfNecessary(Parameters, diagnostics);
```

Comment:
This PR doesn't change local function symbols. Should it?

Reply:
Good catch. Added.